runtime/zeitgeist/src/lib.rs

```diff
@@ -100,6 +102,8 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     state_version: 1,
 };
 
+pub type ContractsCallfilter = Nothing;
```

Comment:
Ok, so no calls allowed for main-net ([here](https://paritytech.github.io/substrate/master/src/frame_support/traits/members.rs.html#83)).